src/scss/style.scss

```diff
@@ -0,0 +1,21 @@
+@import './footer';
+
+
+$color-red: #ff0000;
+
+body {
+  background: lighten($color: $color-red, $amount: 20);
+  font-family: sans-serif;
+  height: 100%;
+  margin: 0;
+  padding: 0;
+  width: 100%;
+}
+
+.welcome {
```

Comment:
I am not sure you used this style anywhere

Reply:
This was the style used in the scss chore as a showcase for what has been implemented.